package.json

```diff
@@ -69,6 +73,7 @@
     "prop-types": "15.7.2",
     "react": "16.10.1",
     "react-dom": "16.10.1",
-    "react-test-renderer": "16.10.1"
+    "react-test-renderer": "16.10.1",
+    "typescript": "^3.7.3"
```

Comment:
I'd prefer to use strict versions like `3.7.3` instead of `^3.7.3` and upgrade manually, sometimes it may break things :)